### ISSUE TITLE
Auditing example for apiserver as a pod can't rotate audit logs

### ISSUE THREAD

**Issue by funkypenguin** [kubernetes/website#30661]
The  examples provided at https://kubernetes.io/docs/tasks/debug-application-cluster/audit/#log-backend re configuring audit logging to a file will prevent the apiserver from ever rotating the audit logs. 

The following hostPath mounts are currently the example:

```
- name: audit
  hostPath:
    path: /etc/kubernetes/audit-policy.yaml
    type: File

- name: audit-log
  hostPath:
    path: /var/log/audit.log
    type: FileOrCreate
```

However, if an individual _file_ is mounted at `/var/log/audit.log`, then the apiserver will error when trying to rotate this file, as follows:

```
2021-11-29T00:29:46.892578169Z stderr F E1129 00:29:46.892498       1 metrics.go:110] Error in audit plugin 'log' affecting 1 audit events: can't rename log file: rename /var/log/kubernetes/audit.log /var/log/kubernetes/audit-2021-11-29T00-29-46.892.log: device or resource busy
```

A better example would be create a directory `/var/log/kubernetes/audit/`, and mount that directory as type `DirectoryOrCreate`, since in this case log rotation would work as expected.

I'm happy to submit a PR to correct this.

**Comment by sftim** [comment]
I'm not sure this is a good first issue @celestehorgan - that'd only be the case where someone was clearly stepping forward to guide a new contributor to pick this fix up, and where we were sure the fix was right.

/help
/sig api-machinery
/language en

**Comment by sftim** [comment]
@funkypenguin please feel very welcome to open that PR!

**Comment by funkypenguin** [comment]
I've raised https://github.com/kubernetes/website/pull/30712, but it felt "too easy" :) Let me know if there's anything else I need to do / explain, or whether more detail is required in the docs themselves!

**Comment by sergeyshevch** [comment]
@funkypenguin @sftim 
Looks like this issue was resolved in #30712 

Maybe we should close this issue?

**Comment by sftim** [comment]
Looks fixed

/close
